packages/dam-app-base/src/AppConfig/AppConfig.tsx

```diff
@@ -185,7 +185,7 @@ export default class AppConfig extends React.Component<Props, State> {
     );
   }
 
-  onParameterChange = (key: string, e: React.ChangeEvent<HTMLInputElement>) => {
+  onParameterChange = (key: string, e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement> | React.ChangeEvent<HTMLSelectElement>) => {
```

Comment:
Same thing here - do we support more elements? I dont see an equivalent change in HTML? I still see the Select tag

Reply:
Type error comes from `TextField` using `onParameterChange` handler:
https://github.com/contentful/apps/pull/336/files/67fad8b43f327d24a5cb9bcfcf8fe2c868d3f577#diff-6a90407ca4cc688bff8f85eecaa40447660585c61bc175d6a1ec7b414c472afaR261

Reply:
Good call. Maybe then removing the `HTMLInputElement`?

Reply:
But this types comes from forma type definition:
https://github.com/contentful/forma-36/blob/master/packages/forma-36-react-components/src/components/TextField/TextField.tsx#L35